web/views.py

```
@@ -3141,24 +3141,30 @@
 @login_required
 @teacher_required
 def message_enrolled_students(request, slug):
```

Ruff (0.8.2)

Missing return type annotation for public function `message_enrolled_students`

(ANN201)